curriculum/challenges/english/25-front-end-development/lecture-importance-of-semantic-html/6729959bf9c8e835f46b3f78.md

```diff
@@ -50,35 +50,35 @@ Think about an element that primarily affects text style to highlight when part
 
 ## --text--
 
-Which HTML element is used to emphasize important text?
+When should you use CSS instead of the `<i>` or `<em>` elements?
```

Comment:
```suggestion
When should you use CSS instead of the `i` or `em` elements?
```

I kinda like the greater than less than, but I noticed that the other two questions don't use them. So I would say to remove them for consistency.